lib/ios/RNNTabBarPresenter.m

```diff
@@ -55,7 +55,7 @@ - (void)mergeOptions:(RNNNavigationOptions *)newOptions currentOptions:(RNNNavig
 	}
 	
 	if (newOptions.bottomTabs.visible.hasValue) {
```

Comment:
Maybe you should also check `newOptions.bottomTabs.animate`.

Reply:
updated